analysis_runner/cromwell.py

```diff
@@ -364,7 +364,9 @@ def _get_wait_interval(
     terminal_statuses = {'succeeded'} | failed_statuses
     status_reported = False
     subprocess.check_output(GCLOUD_ACTIVATE_AUTH, shell=True)
-    url = f'https://cromwell.populationgenomics.org.au/api/workflows/v1/{workflow_id}/status'
+    cromwell_api = 'https://cromwell.populationgenomics.org.au/api/workflows/v1'
```

Comment:
Can you make these URLs constant at the top of the method? Eventually we might want to consider pulling them from a config.

Reply:
Updated this, the URL constant was already being imported 🙃